alvr/client_openxr/src/lib.rs

```diff
@@ -361,6 +361,25 @@ pub fn entry_point() {
     let mut streaming_input_thread = None;
     let views_history = Arc::new(Mutex::new(VecDeque::new()));
 
+    let default_view = xr::View {
+        pose: xr::Posef {
+            orientation: xr::Quaternionf {
+                x: 0.0,
+                y: 0.0,
+                z: 0.0,
+                w: 1.0,
+            },
+            position: xr::Vector3f::default(),
+        },
+        fov: xr::Fovf {
+            angle_left: -0.1,
+            angle_right: 0.1,
+            angle_up: 0.1,
+            angle_down: -0.1,
+        },
+    };
+    let mut last_good_views = vec![ default_view, default_view ];
```

Comment:
This line doesn't look formatted correctly with `cargo fmt` but CI doesn't complain?